data/assets/util/scale_model_helper.asset

```diff
@@ -0,0 +1,111 @@
+local earthAsset = asset.require('scene/solarsystem/planets/earth/earth')
```

Comment:
I don't think this file should exist;  it increases the difficulty for a novice builder to go in and understand and replicate the scale behavior for their own model.  I think the contents of this file should be at the top of the each of the new scale models instead where a new user can exchange the `syncedResource` for a `localResource` and still use the other ones.
It makes it unnecessarily complicated to understand the "educational" assets

Reply:
I don't want to paste 70 lines of code into 10 files just to make their code easier to understand. 

Here "educational" is meant to infer that the assets are used for educating the audience, not educational on how to create assets